sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -453,8 +453,8 @@ class Dataset[T] private[sql](
    * Returns true if this [[Dataset]] contains one or more sources that continuously
    * return data as it arrives. A [[Dataset]] that reads data from a streaming source
    * must be executed as a [[ContinuousQuery]] using the `startStream()` method in
-   * [[DataFrameWriter]].  Methods that return a single answer, (e.g., `count()` or
-   * `collect()`) will throw an [[AnalysisException]] when there is a streaming
+   * [[DataFrameWriter]]. Methods that return a single answer, e.g. `count()` or
```

Comment:
See http://dictionary.cambridge.org/dictionary/english/e-g. There should only be a comma before e.g. not after it.